components/cam/src/physics/cam/scream_abortutils.F90

```diff
@@ -0,0 +1,51 @@
+module scream_abortutils
+
+!-------------------------------------------------
+!Utilities to stop the model in case of
+!catastrophic errors
+!-------------------------------------------------
+implicit none
+private
+
+!public subroutines
+public :: endscreamrun
+
+contains
+
+  subroutine endscreamrun (msg)
+    !-------------------------------------------------
+    ! This subroutine will print the optional message
+    ! received via optional arg "msg" and stop
+    ! the simulation
+    !-------------------------------------------------
+
+#ifdef SPMD
```

Comment:
Where is this macro defined?

Reply:
This is defined during the configuration step when we run E3SM with MPI.

Reply:
Ok, but then this file only works in E3SM builds, not in scream standalone. This file is currently always compiled, so we cannot rely on E3SM macros. Either we make this file compiled only in CIME builds, or we remove deps on E3SM build system macros.